Fri Aug  3 10:53:05 CEST 2018: syslog.warning action 'action 9' suspended, next retry is Fri Aug  3 10:53:34 2018 [v8.16.0 try http://www.rsyslog.com/e/2007 ]
Fri Aug  3 10:53:05 CEST 2018: syslog.err command 'KLogPermitNonKernelFacility' is currently not permitted - did you already set it via a RainerScript command (v6+ config)? [v8.16.0 try http://www.rsyslog.com/e/2222 ]
Fri Aug  3 10:53:05 CEST 2018: syslog.err Could not open output pipe '/dev/xconsole':: No such file or directory [v8.16.0 try http://www.rsyslog.com/e/2039 ]
Fri Aug  3 10:53:08 CEST 2018: mail.err <19>Aug  3 10:53:08 msmtp: host=szerver03.petav.local tls=on auth=on user=host-clients from=root@szerver03.petav.local recipients=root@szerver03.petav.local errormsg='cannot locate host szerver03.petav.local: Temporary failure in name resolution' exitcode=EX_NOHOST
Fri Aug  3 10:53:08 CEST 2018: kern.warning [    0.000000] ACPI: RSDP 0x00000000000F03C0 000024 (v02 INTEL )
Fri Aug  3 10:53:08 CEST 2018: kern.warning [    0.000000] ACPI: XSDT 0x000000007F3E8F10 000044 (v01 INTEL  DG31PR   06222004 MSFT 00010013)
Fri Aug  3 10:53:08 CEST 2018: kern.warning [    0.000000] ACPI: FACP 0x000000007F3E6D90 0000F4 (v04 INTEL  DG31PR   06222004 MSFT 00010013)
Fri Aug  3 10:53:08 CEST 2018: kern.warning [    0.000000] ACPI: DSDT 0x000000007F3E1010 004D33 (v01 INTEL  DG31PR   00000000 INTL 20051117)
Fri Aug  3 10:53:08 CEST 2018: mail.err <19>Aug  3 10:53:08 msmtp: message repeated 2 times: [ host=szerver03.petav.local tls=on auth=on user=host-clients from=root@szerver03.petav.local recipients=root@szerver03.petav.local errormsg='cannot locate host szerver03.petav.local: Temporary failure in name resolution' exitcode=EX_NOHOST]
Fri Aug  3 10:53:08 CEST 2018: mail.err <19>Aug  3 10:53:08 msmtp: host=szerver03.petav.local tls=on auth=on user=host-clients from=root@szerver03.petav.local recipients=root@szerver03.petav.local errormsg='cannot locate host szerver03.petav.local: Temporary failure in name resolution' exitcode=EX_NOHOST
Fri Aug  3 10:53:08 CEST 2018: mail.err <19>Aug  3 10:53:08 msmtp: host=szerver03.petav.local tls=on auth=on user=host-clients from=root@szerver03.petav.local recipients=root@szerver03.petav.local errormsg='cannot locate host szerver03.petav.local: Temporary failure in name resolution' exitcode=EX_NOHOST
Fri Aug  3 10:53:08 CEST 2018: kern.warning [    0.000000] ACPI: FACS 0x000000007F3EBF40 000040
Fri Aug  3 10:53:08 CEST 2018: mail.err <19>Aug  3 10:53:08 msmtp: host=szerver03.petav.local tls=on auth=on user=host-clients from=root@szerver03.petav.local recipients=root@szerver03.petav.local errormsg='cannot locate host szerver03.petav.local: Temporary failure in name resolution' exitcode=EX_NOHOST
Fri Aug  3 10:53:08 CEST 2018: kern.warning [    0.000000] ACPI: FACS 0x000000007F3E7F40 000040
Fri Aug  3 10:53:08 CEST 2018: mail.err <19>Aug  3 10:53:08 msmtp: host=szerver03.petav.local tls=on auth=on user=host-clients from=root@szerver03.petav.local recipients=root@szerver03.petav.local errormsg='cannot locate host szerver03.petav.local: Temporary failure in name resolution' exitcode=EX_NOHOST
Fri Aug  3 10:53:08 CEST 2018: kern.warning [    0.000000] ACPI: APIC 0x000000007F3E6F10 00006C (v02 INTEL  DG31PR   06222004 MSFT 00010013)
Fri Aug  3 10:53:08 CEST 2018: kern.warning [    0.000000] ACPI: HPET 0x000000007F3EFD10 000038 (v01 INTEL  ICH7HPET 06222004 AMI. 00000001)
Fri Aug  3 10:53:08 CEST 2018: mail.err <19>Aug  3 10:53:08 msmtp: host=szerver03.petav.local tls=on auth=on user=host-clients from=root@szerver03.petav.local recipients=root@szerver03.petav.local errormsg='cannot locate host szerver03.petav.local: Temporary failure in name resolution' exitcode=EX_NOHOST
Fri Aug  3 10:53:08 CEST 2018: kern.warning [    0.000000] ACPI: MCFG 0x000000007F3EFC10 00003C (v01 INTEL  DG31PR   06222004 MSFT 00000097)
Fri Aug  3 10:53:08 CEST 2018: kern.warning [    0.010156] ACPI: 1 ACPI AML tables successfully acquired and loaded
Fri Aug  3 10:53:08 CEST 2018: mail.err <19>Aug  3 10:53:08 msmtp: message repeated 2 times: [ host=szerver03.petav.local tls=on auth=on user=host-clients from=root@szerver03.petav.local recipients=root@szerver03.petav.local errormsg='cannot locate host szerver03.petav.local: Temporary failure in name resolution' exitcode=EX_NOHOST]
Fri Aug  3 10:53:08 CEST 2018: kern.warning [    0.112008] PCCT header not found.
Fri Aug  3 10:53:08 CEST 2018: mail.err <19>Aug  3 10:53:08 msmtp: host=szerver03.petav.local tls=on auth=on user=host-clients from=root@szerver03.petav.local recipients=root@szerver03.petav.local errormsg='cannot locate host szerver03.petav.local: Temporary failure in name resolution' exitcode=EX_NOHOST
Fri Aug  3 10:53:08 CEST 2018: mail.err <19>Aug  3 10:53:08 msmtp: host=szerver03.petav.local tls=on auth=on user=host-clients from=root@szerver03.petav.local recipients=root@szerver03.petav.local errormsg='cannot locate host szerver03.petav.local: Temporary failure in name resolution' exitcode=EX_NOHOST
Fri Aug  3 10:53:08 CEST 2018: kern.warning [    0.125573] ACPI: Executed 1 blocks of module-level executable AML code
Fri Aug  3 10:53:08 CEST 2018: mail.err <19>Aug  3 10:53:08 msmtp: host=szerver03.petav.local tls=on auth=on user=host-clients from=root@szerver03.petav.local recipients=root@szerver03.petav.local errormsg='cannot locate host szerver03.petav.local: Temporary failure in name resolution' exitcode=EX_NOHOST
Fri Aug  3 10:53:08 CEST 2018: kern.warning [    0.126820] ACPI: Dynamic OEM Table Load:
Fri Aug  3 10:53:08 CEST 2018: kern.warning [    0.126826] ACPI: SSDT 0xFFFF88007C33D800 0002CC (v01 AMI    IST      00000001 MSFT 03000001)
Fri Aug  3 10:53:08 CEST 2018: kern.warning [    0.136589] ACPI: Enabled 3 GPEs in block 00 to 1F
Fri Aug  3 10:53:08 CEST 2018: mail.err <19>Aug  3 10:53:08 msmtp: message repeated 3 times: [ host=szerver03.petav.local tls=on auth=on user=host-clients from=root@szerver03.petav.local recipients=root@szerver03.petav.local errormsg='cannot locate host szerver03.petav.local: Temporary failure in name resolution' exitcode=EX_NOHOST]
Fri Aug  3 10:53:08 CEST 2018: kern.warning [    7.955355] zavl: loading out-of-tree module taints kernel.
Fri Aug  3 10:53:08 CEST 2018: kern.warning [    7.955360] zavl: module license 'CDDL' taints kernel.
Fri Aug  3 10:53:08 CEST 2018: mail.err <19>Aug  3 10:53:08 msmtp: host=szerver03.petav.local tls=on auth=on user=host-clients from=root@szerver03.petav.local recipients=root@szerver03.petav.local errormsg='cannot locate host szerver03.petav.local: Temporary failure in name resolution' exitcode=EX_NOHOST
Fri Aug  3 10:53:08 CEST 2018: kern.warning [    7.955361] Disabling lock debugging due to kernel taint
Fri Aug  3 10:53:08 CEST 2018: kern.warning [    8.542781] SPL: The /etc/hostid file is not found.
Fri Aug  3 10:53:08 CEST 2018: mail.err <19>Aug  3 10:53:08 msmtp: message repeated 2 times: [ host=szerver03.petav.local tls=on auth=on user=host-clients from=root@szerver03.petav.local recipients=root@szerver03.petav.local errormsg='cannot locate host szerver03.petav.local: Temporary failure in name resolution' exitcode=EX_NOHOST]
Fri Aug  3 10:53:08 CEST 2018: user.err <11>Aug  3 10:53:08 mcelog: failed to prefill DIMM database from DMI data
Fri Aug  3 10:53:08 CEST 2018: mail.err <19>Aug  3 10:53:08 msmtp: host=szerver03.petav.local tls=on auth=on user=host-clients from=root@szerver03.petav.local recipients=root@szerver03.petav.local errormsg='cannot locate host szerver03.petav.local: Temporary failure in name resolution' exitcode=EX_NOHOST
Fri Aug  3 10:53:08 CEST 2018: kern.warning [   20.778413] intel_rng: Firmware space is locked read-only. If you can't or
Fri Aug  3 10:53:08 CEST 2018: kern.warning [   20.778413] intel_rng: don't want to disable this in firmware setup, and if
Fri Aug  3 10:53:08 CEST 2018: kern.warning [   20.778413] intel_rng: you are certain that your system has a functional
Fri Aug  3 10:53:08 CEST 2018: kern.warning [   20.778413] intel_rng: RNG, try using the 'no_fwh_detect' option.
Fri Aug  3 10:53:08 CEST 2018: daemon.warning <28>Aug  3 10:53:00 systemd-tmpfiles[1657]: [/usr/lib/tmpfiles.d/var.conf:14] Duplicate line for path "/var/log", ignoring.
Fri Aug  3 10:53:08 CEST 2018: daemon.warning <28>Aug  3 10:53:03 zed: eid=1 class=statechange
Fri Aug  3 10:53:08 CEST 2018: daemon.warning <28>Aug  3 10:53:03 zed: eid=2 class=config.sync pool=rootpool
Fri Aug  3 10:53:08 CEST 2018: daemon.warning <28>Aug  3 10:53:03 zed: eid=3 class=config.sync pool=rootpool
Fri Aug  3 10:53:08 CEST 2018: mail.err <19>Aug  3 10:53:08 msmtp: message repeated 7 times: [ host=szerver03.petav.local tls=on auth=on user=host-clients from=root@szerver03.petav.local recipients=root@szerver03.petav.local errormsg='cannot locate host szerver03.petav.local: Temporary failure in name resolution' exitcode=EX_NOHOST]
Fri Aug  3 10:53:10 CEST 2018: kern.warning [   36.274588] audit_printk_skb: 24 callbacks suppressed
Fri Aug  3 10:53:10 CEST 2018: mail.err <19>Aug  3 10:53:10 msmtp: host=szerver03.petav.local tls=on auth=on user=host-clients from=root@szerver03.petav.local recipients=root@szerver03.petav.local errormsg='cannot locate host szerver03.petav.local: Name or service not known' exitcode=EX_NOHOST
Fri Aug  3 10:53:11 CEST 2018: kern.warning <4>Aug  3 10:53:11 NetworkManager[1948]: <warn>  [1533286391.3413] SettingsPlugin-Ofono: file doesn't exist: /var/lib/ofono
Fri Aug  3 10:53:11 CEST 2018: mail.err <19>Aug  3 10:53:11 msmtp: host=szerver03.petav.local tls=on auth=on user=host-clients from=root@szerver03.petav.local recipients=root@szerver03.petav.local errormsg='cannot locate host szerver03.petav.local: Name or service not known' exitcode=EX_NOHOST
Fri Aug  3 10:53:12 CEST 2018: kern.err <3>Aug  3 10:53:12 NetworkManager[1948]: nm_device_get_device_type: assertion 'NM_IS_DEVICE (self)' failed
Fri Aug  3 10:53:12 CEST 2018: kern.warning <4>Aug  3 10:53:12 NetworkManager[1948]: <warn>  [1533286392.9015] failed to enumerate oFono devices: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.ofono was not provided by any .service files
Fri Aug  3 10:53:12 CEST 2018: mail.err <19>Aug  3 10:53:12 msmtp: host=szerver03.petav.local tls=on auth=on user=host-clients from=root@szerver03.petav.local recipients=root@szerver03.petav.local errormsg='cannot locate host szerver03.petav.local: Name or service not known' exitcode=EX_NOHOST
Fri Aug  3 10:53:16 CEST 2018: daemon.warning <28>Aug  3 10:53:16 dnsmasq[2363]: warning: no upstream servers configured
Fri Aug  3 10:53:16 CEST 2018: mail.err <19>Aug  3 10:53:16 msmtp: host=szerver03.petav.local tls=on auth=on user=host-clients from=root@szerver03.petav.local recipients=root@szerver03.petav.local errormsg='cannot locate host szerver03.petav.local: Name or service not known' exitcode=EX_NOHOST
Fri Aug  3 10:53:17 CEST 2018: authpriv.err <83>Aug  3 10:53:17 lightdm: PAM unable to dlopen(pam_kwallet.so): /lib/security/pam_kwallet.so: cannot open shared object file: No such file or directory
Fri Aug  3 10:53:17 CEST 2018: authpriv.err <83>Aug  3 10:53:17 lightdm: PAM adding faulty module: pam_kwallet.so
Fri Aug  3 10:53:17 CEST 2018: authpriv.err <83>Aug  3 10:53:17 lightdm: PAM unable to dlopen(pam_kwallet5.so): /lib/security/pam_kwallet5.so: cannot open shared object file: No such file or directory
Fri Aug  3 10:53:17 CEST 2018: authpriv.err <83>Aug  3 10:53:17 lightdm: PAM adding faulty module: pam_kwallet5.so
Fri Aug  3 10:53:25 CEST 2018: user.warning <12>Aug  3 10:53:25 org.gnome.ScreenSaver[2500]: ** (gnome-screensaver:2611): WARNING **: Couldn't get presence status: The name org.gnome.SessionManager was not provided by any .service files
Fri Aug  3 10:53:25 CEST 2018: mail.err <19>Aug  3 10:53:25 msmtp: host=szerver03.petav.local tls=on auth=on user=host-clients from=root@szerver03.petav.local recipients=root@szerver03.petav.local errormsg='cannot locate host szerver03.petav.local: Name or service not known' exitcode=EX_NOHOST
Fri Aug  3 10:53:27 CEST 2018: authpriv.err <83>Aug  3 10:53:27 lightdm: PAM unable to dlopen(pam_kwallet.so): /lib/security/pam_kwallet.so: cannot open shared object file: No such file or directory
Fri Aug  3 10:53:27 CEST 2018: authpriv.err <83>Aug  3 10:53:27 lightdm: PAM adding faulty module: pam_kwallet.so
Fri Aug  3 10:53:27 CEST 2018: authpriv.err <83>Aug  3 10:53:27 lightdm: PAM unable to dlopen(pam_kwallet5.so): /lib/security/pam_kwallet5.so: cannot open shared object file: No such file or directory
Fri Aug  3 10:53:27 CEST 2018: authpriv.err <83>Aug  3 10:53:27 lightdm: PAM adding faulty module: pam_kwallet5.so
Fri Aug  3 10:53:33 CEST 2018: user.warning <12>Aug  3 10:53:33 org.gtk.vfs.Daemon[2500]: A connection to the bus can't be made
Fri Aug  3 10:53:33 CEST 2018: mail.err <19>Aug  3 10:53:33 msmtp: host=szerver03.petav.local tls=on auth=on user=host-clients from=root@szerver03.petav.local recipients=root@szerver03.petav.local errormsg='cannot locate host szerver03.petav.local: Name or service not known' exitcode=EX_NOHOST
Fri Aug  3 10:53:36 CEST 2018: user.warning <12>Aug  3 10:53:36 org.a11y.Bus[3036]: ** (process:3150): WARNING **: Failed to register client: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.gnome.SessionManager was not provided by any .service files
Fri Aug  3 10:53:36 CEST 2018: mail.err <19>Aug  3 10:53:36 msmtp: host=szerver03.petav.local tls=on auth=on user=host-clients from=root@szerver03.petav.local recipients=root@szerver03.petav.local errormsg='cannot locate host szerver03.petav.local: Name or service not known' exitcode=EX_NOHOST
Fri Aug  3 10:53:36 CEST 2018: user.warning <12>Aug  3 10:53:36 org.a11y.Bus[3036]: Activating service name='org.a11y.atspi.Registry'
Fri Aug  3 10:53:36 CEST 2018: user.warning <12>Aug  3 10:53:36 org.a11y.Bus[3036]: Successfully activated service 'org.a11y.atspi.Registry'
Fri Aug  3 10:53:36 CEST 2018: mail.err <19>Aug  3 10:53:36 msmtp: host=szerver03.petav.local tls=on auth=on user=host-clients from=root@szerver03.petav.local recipients=root@szerver03.petav.local errormsg='cannot locate host szerver03.petav.local: Name or service not known' exitcode=EX_NOHOST
Fri Aug  3 10:53:40 CEST 2018: mail.err <19>Aug  3 10:53:36 msmtp: host=szerver03.petav.local tls=on auth=on user=host-clients from=root@szerver03.petav.local recipients=root@szerver03.petav.local errormsg='cannot locate host szerver03.petav.local: Name or service not known' exitcode=EX_NOHOST
Fri Aug  3 10:53:40 CEST 2018: authpriv.notice <85>Aug  3 10:53:40 polkitd(authority=local): Registered Authentication Agent for unix-session:c2 (system bus name :1.33 [/usr/bin/lxsession -s Lubuntu -e LXDE], object path /org/freedesktop/PolicyKit1/AuthenticationAgent, locale en_US.UTF-8)
Fri Aug  3 10:53:49 CEST 2018: auth.notice <37>Aug  3 10:53:49 dbus[1861]: [system] Failed to activate service 'org.bluez': timed out
Fri Aug  3 10:53:50 CEST 2018: authpriv.notice <85>Aug  3 10:53:50 sudo: rendszergazda : TTY=pts/0 ; PWD=/home/rendszergazda ; USER=root ; COMMAND=/usr/bin/mc
